src-built-in/adapters/indexedDBAdapter.js

```diff
@@ -147,6 +152,7 @@ const IndexedDBAdapter = function () {
 	 * @private
 	 */
 	this.releaseQueue = () => {
+		initialized = true;
```

Comment:
@kriswest, forgive me if this is a dumb question; a lot of this is still opaque to me. What's the connection between `this.releaseQueue` being called and the db's initialization? How do you know it's finished by by the time this is called?

Reply:
Looks like the difference is if it follows the "onupgradeneeded" path. in this case it only gets initialized after the oncomplete as opposed to the previous case where this.db was getting set before that.

Reply:
Duh, race condition. Good catch.